cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -107,7 +107,7 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
     for ( var i = 0, len = list.length; i < len; i++ ) {
       item = list[i];
 
-      // If the value isn't valid kill the script and propt the developer
+      // If the value isn't valid kill the script and prompt the developer.
       if ( !strings.stringValid( item.value ) ) {
         console.log( '\'' + item.value + '\' is not a valid value' );
```

Comment:
NOOOOO! The problem was that errors were stopping all JS on the page, which is why the checker was added in the first place.